src/cmd/generate.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/nasz-elektryk/spito/checker"
+	"github.com/nasz-elektryk/spito/cmd/cmdApi"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+const exampleRuleContents = `function main()
+  return true
+end
+`
+
+func handleGenerate(cmd *cobra.Command, args []string) {
+	configFileContents, error := os.ReadFile(checker.CONFIG_FILENAME)
+	if os.IsNotExist(error) {
+		printErrorAndExit(errors.New("Please run this commnd inside an actual spito ruleset directory!"))
+	}
+	handleError(error)
+
+	config := ConfigFileLayout{}
+	error = yaml.Unmarshal(configFileContents, &config)
+	handleError(error)
+
+	rulesetLocation := checker.RuleSetLocation{}
+	rulesetLocation.New(config.Identifier)
+	if !rulesetLocation.IsRuleSetDownloaded() {
+		printErrorAndExit(errors.New("Your ruleset is not in the standard ruleset path!"))
+	}
+
+	ruleName := strings.ReplaceAll(args[0], " ", "")
+	ruleFile, error := os.Create("rules/" + ruleName + ".lua")
```

Comment:
I think it should be possible to create new rule in subdirectory e.g. create rule in `rules/example/dir/rule.lua`